docs/internals/deprecation.txt

```diff
@@ -30,6 +30,12 @@ details on these changes.
 
 * ``Field.remote_field.to`` attribute will be removed.
 
+* The ``ForeignKey.on_delete`` argument will be required.
```

Comment:
one bullet point is enough: The on_delete argument for ForeignKey, ForeignObject, and OneToOneField ....